presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto server can be configured to enable Kerberos authentication over
+HTTPS for clients such as the :doc:`Presto CLI </security/cli>` or the various
+JDBC and ODBC drivers.
+
+.. warning::
+
+   Unauthenticated access over HTTP *must* be enabled because worker nodes
+   cannot yet be configured to connect to the coordinator using HTTPS or to
+   authenticate with Kerberos. It is the adminstrator's responsibility to
+   ensure that insecure access to the Presto coordinator is blocked for any
+   node that isn't a worker node.  This can be done by blocking access to the
+   Presto coordinator's HTTP port from nodes that are not workers.
+
```

Comment:
Suggest the following for the warning (it starts with a negative, which I think is okay in this case for pointing out an important negative).

Worker nodes cannot yet be configured to connect to the Presto coordinator using HTTPS or to authenticate with Kerberos. It is the administrator's responsibility to enable unauthenticated access over HTTP for worker nodes and ensure unathenticated access is blocked for any node that is not a worker node. This can be done by blocking access to the Presto coordinator's HTTP port from nodes that are not workers.